modules/apps/collaboration/wiki/wiki-editor-configuration/src/test/java/com/liferay/wiki/editor/configuration/internal/WikiAttachmentImageHTMLEditorConfigContributorTest.java

```diff
@@ -165,6 +170,11 @@ public void testItemSelectorURLWhenAllowBrowseAndValidWikiPage()
 		wikiAttachmentImageHTMLEditorConfigContributor.setItemSelector(
 			_itemSelector);
 
+		suppress(
```

Comment:
That's what I thought at first sight too, but the test failed (https://github.com/pei-jung/liferay-portal/pull/187#issuecomment-297830908). When I traced down the error, it turns out that it's trying to call the new method. 

in the test, we call `wikiAttachmentImageHTMLEditorConfigContributor.populateConfigJSONObject`, which calls `getItemSelectorURL` and triggers `getUploadItemSelectorCriterion` that tries to create a new instance of `UploadItemSelectorCriterion`. Since I replaced the props usage with the new method in 1d5c095aeb5713de7e475cca46d2a00ce5e4c931, the test complains that it couldn't find the new method. And looking at the test, it seems it doesn't depend on `getUploadItemSelectorCriterion`, so I suppress it.